components/board/upboard/board.go

```diff
@@ -0,0 +1,31 @@
+// Package upboard implements an Intel based board.
+package upboard
+
+/*
+	Datasheet: https://github.com/up-board/up-community/wiki/Pinout_UP4000
+	Supported board: UP4000
```

Comment:
I think this is supposed to go above the package name declaration, between lines 1 and 2.

Reply:
Following the convention we agreed upon. Same pattern exists in uln and imu code.

Reply:
My mistake!